packages/core/src/Stores/notification-store.js

```diff
@@ -1078,6 +1082,16 @@ export default class NotificationStore extends BaseStore {
                 message: <Localize i18n_default_text='Please log in with your updated password.' />,
                 type: 'info',
             },
+            phone_number_verification: {
+                key: 'phone_number_verification',
+                header: localize('Verify your phone number'),
+                message: <Localize i18n_default_text='Keep your account safe. Verify your phone number now.' />,
+                type: 'warning',
+                action: {
+                    route: routes.phone_verification,
+                    text: localize('Get started'),
+                },
+            },
```

Comment:
We need to add it to in packages/core/src/App/Containers/app-notification-messages.jsx#L92 for showing in mobile